lib/matplotlib/ticker.py

```diff
@@ -416,15 +417,17 @@ class ScalarFormatter(Formatter):
     axes.formatter.limits rc parameter.
 
     """
+    _usingOffset = False
+    _usingScaling = False
 
     def __init__(self, useOffset=None, useMathText=None, useLocale=None):
         # useOffset allows plotting small data ranges with large offsets: for
         # example: [1+1e-9,1+2e-9,1+3e-9] useMathText will render the offset
         # and scientific notation in mathtext
 
-        if useOffset is None:
-            useOffset = rcParams['axes.formatter.useoffset']
-        self.set_useOffset(useOffset)
+        self.set_useOffset(False)
```

Comment:
The practice in the rest of the library is that the value of the rcparams are stashed when the artists are created.  In part this is so that the rcparams context manager and in part so that we can be sure of _when_ the rcprams are consulted (many things are deffered to draw time).